white/continuously-improve-your-security-belt-rank.md

```diff
@@ -1,12 +1,17 @@
 # Continuously Improve your Security Belt Rank
 
-Security is like a big pizza. You cannot eat it as a whole, but you can slice it and continuously eat small slices. To make this happen, ensure that the Product Owner continuously prioritizes the security belt activities for the next belt highly within the product backlog. Security belt activities make good slices because they are of reasonable size and have a defined output. Celebrate all your implemented security belt activities.
+Security is like a big pizza. You cannot eat it as a whole, but you can slice it and continuously eat small slices.
+
+To make this happen:
+- ensure that the Product Owner continuously prioritizes the security belt activities for the next belt highly within the product backlog.
+- Celebrate all your implemented security belt activities.
```

Comment:
```suggestion
Security is like a big pizza. You cannot eat it as a whole, but you can slice it and continuously eat small slices. Security belt activities make good slices because they are of reasonable size and have a defined output.

To make this happen:
- ensure that the Product Owner continuously prioritizes the security belt activities for the next belt highly within the product backlog.
- Celebrate all your implemented security belt activities.
```